cluster-scope/overlays/nerc-ocp-infra/machineconfigs/configure-bond0/Makefile

```diff
@@ -0,0 +1,10 @@
+MACHINECONFIGS = \
```

Comment:
Where is this makefile being used? Maybe I'm missing where it runs

Reply:
You run it locally if you make changes to anything in the `src/` directory.